google-cloud-tools-plugin/ultimate/build.gradle

```diff
@@ -39,6 +39,7 @@ dependencies {
     compile ('com.google.cloud.tools:appengine-plugins-core:' + toolsLibVersion) {
         exclude group: 'com.google.guava', module: 'guava'
     }
+    compile 'org.yaml:snakeyaml:1.18'
```

Comment:
two things then: can you make it testCompile instead of compile?

or better yet, split up the test so that you separate the community from the ultimate parts? that way you don't have to add the dependency just for testing.